docs/node-operators/snark-workers/getting-started.mdx

```diff
@@ -1,21 +1,87 @@
 ---
-title: Snark Workers
+title: Getting Started
 hide_title: true
 description: How to run a Snark Worker. Snark Workers are essential to keep the network producing transactions in blocks. 
```

Comment:
```suggestion
description: How to run a Snark Worker. Snark Workers are essential to keep the network producing transactions in blocks. 
```
```suggestion
description: How to run a SNARK worker. SNARK workers are essential to keep the network producing transactions in blocks. 
```

Reply:
SNARK is always all upper-case see https://docs.minaprotocol.com/glossary#snark-worker